wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java

```diff
@@ -122,6 +122,196 @@ public void test_transfer_with_fee() throws IOException, ErrorResponseException
+        }
+        fail();
+    }
+
+    @Test
+    public void test_missing_account() {
+        Neow3j neow3j = ResponseInterceptor.createNeow3jWithInceptor(null);
+        RawTransactionOutput output = new RawTransactionOutput(NEOAsset.HASH_ID, "1", "AJQ6FoaSXDFzA6wLnyZ1nFN7SGSN2oNTc3");
+
+        try {
+            AssetTransfer at = new AssetTransfer.Builder()
+                    .neow3j(neow3j)
+                    .output(output)
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
+    public void test_missing_outputs() {
+        Neow3j neow3j = ResponseInterceptor.createNeow3jWithInceptor(null);
+        Account a = Account.fromWIF("KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr").build();
+
+        try {
+            AssetTransfer at = new AssetTransfer.Builder()
+                    .neow3j(neow3j)
+                    .account(a)
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
+    public void test_partially_missing_output1() {
+        Neow3j neow3j = ResponseInterceptor.createNeow3jWithInceptor(null);
+        Account a = Account.fromWIF("KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr").build();
+
+        try {
+            AssetTransfer at = new AssetTransfer.Builder()
+                    .neow3j(neow3j)
+                    .account(a)
+                    .asset(NEOAsset.HASH_ID)
+                    .amount(BigDecimal.ONE)
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
+    public void test_partially_missing_output2() {
+        Neow3j neow3j = ResponseInterceptor.createNeow3jWithInceptor(null);
+        Account a = Account.fromWIF("KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr").build();
+
+        try {
+            AssetTransfer at = new AssetTransfer.Builder()
+                    .neow3j(neow3j)
+                    .account(a)
+                    .asset(NEOAsset.HASH_ID)
+                    .toAddress("AJQ6FoaSXDFzA6wLnyZ1nFN7SGSN2oNTc3")
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
+    public void test_partially_missing_output3() {
+        Neow3j neow3j = ResponseInterceptor.createNeow3jWithInceptor(null);
+        Account a = Account.fromWIF("KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr").build();
+
+        try {
+            AssetTransfer at = new AssetTransfer.Builder()
+                    .neow3j(neow3j)
+                    .account(a)
+                    .amount(BigDecimal.ONE)
+                    .toAddress("AJQ6FoaSXDFzA6wLnyZ1nFN7SGSN2oNTc3")
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
```

Comment:
Can you please adapt all the tests that you expect an exception with:
@Test(expect = SpecificException.class)